database_admin/schema/create_schema.sql

```diff
@@ -766,7 +766,6 @@ CREATE TABLE IF NOT EXISTS advisory_metadata
     cve_list         JSONB,
     reboot_required  BOOLEAN NOT NULL DEFAULT false,
     release_versions JSONB,
-    synced           BOOLEAN NOT NULL DEFAULT false,
```

Comment:
Michal please be careful when rebasing. this line has been added to master by (my) latest PR.